docs/CONFIG-VARS.md

```diff
@@ -227,25 +227,61 @@ stateful = {
-| postgres_ssl_enforcement_enabled | Enforce SSL on connection to the PostgreSQL database | bool | false | |
-| postgres_db_charset | Charset for the PostgreSQL Database | string | "UTF8" | Needs to be a valid PostgreSQL Charset. |
-| postgres_db_collation | Collation for the PostgreSQL Database | string | "en_US.UTF8" | Needs to be a valid PostgreSQL Collation. |
-| postgres_backups_enabled | Enables postgres backups | bool | true | |
-| postgres_backups_start_time | Start time for postgres backups | string | "21:00" | |
-| postgres_backups_location | TODO | string | null | |
-| postgres_backups_point_in_time_recovery_enabled | Enable point-in-time recovery | bool | false | |
-| postgres_db_names | The list of names of PostgreSQL database to create | list | [] | |
-| postgres_availability_type | The availability type for the master instance. | string | "ZONAL" | This is only used to set up high availability for the PostgreSQL instance. Can be either `ZONAL` or `REGIONAL`."
-| postgres_database_flags | Database flags for the master instance. | list of objects |  | More details: https://cloud.google.com/sql/docs/postgres/flags |
+| machine_type| The machine type for the PostgreSQL server VMs" | string | "db-custom-8-30720" | Google Cloud Postgres supports only shared-core machine types such as db-f1-micro, and custom machine types such as db-custom-2-13312.
+| storage_gb | Minimum storage allowed for the PostgreSQL server | number | 10 | |
+| backups_enabled | Enables postgres backups | bool | true | |
+| backups_start_time | Start time for postgres backups | string | "21:00" | |
+| backups_location | TODO | string | null | |
+| backups_point_in_time_recovery_enabled | Enable point-in-time recovery | bool | false | |
+| administrator_login | The Administrator Login for the PostgreSQL Server. Changing this forces a new resource to be created. | string | "pgadmin" | | |
+| administrator_password | The Password associated with the administrator_login for the PostgreSQL Server | string | "my$up3rS3cretPassw0rd" |  |
+| server_version | The version of the  PostgreSQL server instance | string | "11" | Supported values are 11 and 12 |
+| ssl_enforcement_enabled | Enforce SSL on connection to the PostgreSQL database | bool | true | |
+| availability_type | The availability type for the master instance. | string | "ZONAL" | This is only used to set up high availability for the PostgreSQL instance. Can be either `ZONAL` or `REGIONAL`." |
+| database_flags | Database flags for the master instance. | list of objects |  | More details can be found [here](https://cloud.google.com/sql/docs/postgres/flags) |
+
+Here is a sample of the `postgres_servers` variable with the `default` entry only overriding the `administrator_password` parameter and the `cps` entry overriding all of the parameters:
+
+```terraform
+postgres_servers = {
+  default = {
```

Comment:
The CPS example has any additional level. The default example appears to be missing one.

Reply:
This has been fixed.